starter-code/app.js

```diff
@@ -3,3 +3,34 @@ const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+
+//configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+//routes
```

Comment:
It is a very good idea to have a route for /, if you want to display the same thing that in /random you can do:
```
app.get("/", (req, res) => {
  res.redirect('/random');
});
```